### PR TITLE
Parameterized defaultdict

### PR DESCRIPTION
**Pull request by st0le**

### CODE REVIEW

File: keydefaultdict.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            raise KeyError( key )
+        else:
+            ret = self[key] = self.default_factory(key)
+            return ret
+
+def pow2(n):
+	return 1 << n 
+
+d = keydefaultdict(pow2)
+print d[1]
```
Comment (brennerm) [inline keydefaultdict.py]: Please use print()

File: keydefaultdict.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+print d[3]
```
Comment (brennerm) [inline keydefaultdict.py]: Please use print()

File: keydefaultdict.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+print d[10]
```
Comment (brennerm) [inline keydefaultdict.py]: Please use print()

File: keydefaultdict.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+print d
```
Comment (brennerm) [inline keydefaultdict.py]: Please use print()

**Comment by st0le** [comment]
```
converted print statements to print()
```

**Comment by brennerm** [comment]
Thanks :)

**Pull request merged**